render.js

```diff
@@ -0,0 +1,55 @@
+const cls = $('#close')
```

Comment:
better to just name it `close` no reason to make it shorter. It just makes it more difficult to read